pkg/storageops/vsphere/vsphere_util.go

```diff
@@ -80,10 +80,7 @@ func ReadVSphereConfigFromEnv() (*VSphereConfig, error) {
 		cfg.InsecureFlag = true
 	}
 
-	cfg.VMUUID, err = storageops.GetEnvValueStrict("VSPHERE_VM_UUID")
-	if err != nil {
-		return nil, err
-	}
+	cfg.VMUUID, _ = storageops.GetEnvValueStrict("VSPHERE_VM_UUID")
```

Comment:
curious why this change is needed?

Reply:
VMUUID is required to be fetched from the env variable`VSPHERE_VM_UUID` only if one is running unit tests for dev.

When running in a real vSphere environment, Portworx (porx) will fetch this from a cloudprovider field that k8s populates in the node object. So we don't need this env. For e.g

```
- apiVersion: v1
  kind: Node
  metadata:
    creationTimestamp: 2018-08-13T03:46:12Z
  spec:
    externalID: 421076c2-e509-7897-e64f-84b0bff79cbe
    providerID: vsphere://421076c2-e509-7897-e64f-84b0bff79cbe
```